deploy/examples/das-operator.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: inference.redhat.com/v1alpha1
+kind: DASOperator
+metadata:
+  name: cluster
+  namespace: das-operator
+spec:
+  emulatedMode: disabled
```

Comment:
I wavered on whether to enable emulated mode and am very open to being corrected here

Reply:
All production use cases should have emulatedMode `disabled`.  It is only useful for development and somewhat in CI.